functions/source/soci-index-generator-lambda/utils/registry/registry.go

```diff
@@ -116,9 +130,9 @@ func (registry *Registry) HeadManifest(ctx context.Context, repositoryName strin
 	return descriptor, nil
 }
 
-// Call registry's getManifest and return the image's manifest
+// Call registry's FetchReference and return the image's reference
 // The image reference must be a digest because that's what oras-go FetchReference takes
-func (registry *Registry) GetManifest(ctx context.Context, repositoryName string, digest string) (ocispec.Manifest, error) {
+func (registry *Registry) GetDigest(ctx context.Context, repositoryName string, digest string) (ocispec.Manifest, error) {
```

Comment:
The previous comment and function name seems to match what this function does. Why did this change it?